util/libeay.num

```diff
@@ -4370,7 +4370,7 @@ DH_compute_key_padded                   4732	EXIST::FUNCTION:DH
 ECDSA_METHOD_set_sign                   4733	EXIST::FUNCTION:ECDSA
 CMS_RecipientEncryptedKey_cert_cmp      4734	EXIST:!VMS:FUNCTION:CMS
 CMS_RecipEncryptedKey_cert_cmp          4734	EXIST:VMS:FUNCTION:CMS
-DH_KDF_X9_42                            4735	EXIST::FUNCTION:DH
+DH_KDF_X9_42                            4735	EXIST::FUNCTION:CMS,DH
```

Comment:
So you mean that there would be a packager that suddenly, just because it becomes available, would make packages with `no-cms` when the earlier packages of that openssl version series were configured without it? 

I'm sorry, but I can't see how that's our responsibility, much more than a packager suddenly configuring `no-rsa` mid-series.